pynautobot/core/app.py

```diff
@@ -82,16 +82,18 @@ def choices(self):
 
         return self._choices
 
-    def custom_choices(self):
+    def custom_fields(self):
         """Returns custom-fields response from app
 
         :Returns: Raw response from Nautobot's custom-fields endpoint.
         :Raises: :py:class:`.RequestError` if called for an invalid endpoint.
         :Example:
 
-        >>> nb.extras.custom_choices()
-        {'Testfield1': {'Testvalue2': 2, 'Testvalue1': 1},
-         'Testfield2': {'Othervalue2': 4, 'Othervalue1': 3}}
+        >>> custom_fields_list = nb.extras.custom_fields()
+        >>> print(custom_fields_list[0]['label'])
+        Test custom field for rack
+        >>> print(custom_fields_list[0]['content_types'])
+        ['dcim.rack']
```

Comment:
```suggestion
        >>> nb.extras.custom_fields()
        [
            {
                "id": "5b39ba88-e5ab-4be2-89f5-5a016473b53c",
                "display": "Test custom field",
                "url": "http://localhost:8000/api/extras/custom-fields/5b39ba88-e5ab-4be2-89f5-5a016473b53c/",
                "content_types": ["dcim.rack"],
                "type": {"value": "integer", "label": "Integer"},
                "label": "Test custom field",
                "name": "test_custom_field",
                "slug": "test_custom_field",
                "description": "",
                "required": False,
                "filter_logic": {"value": "loose", "label": "Loose"},
                "default": None,
                "weight": 100,
                "validation_minimum": None,
                "validation_maximum": None,
                "validation_regex": "",
                "created": "2023-04-15",
                "last_updated": "2023-04-15T17:45:11.839431Z",
                "notes_url": "http://localhost:8000/api/extras/custom-fields/5b39ba88-e5ab-4be2-89f5-5a016473b53c/notes/",
            },
        ]
```
Examples should show what is really returned from the method.
Think about someone who is looking at this for the first time.